deploy/deployment.yaml

```diff
@@ -17,9 +17,26 @@ spec:
         application: trivy-dojo-report-operator
     spec:
       serviceAccountName: trivy-dojo-report-account
+      podSecurityContext:
+        enabled: true
+        fsGroupChangePolicy: Always
+        sysctls: []
+        supplementalGroups: []
+        fsGroup: 1001
       containers:
         - name: trivy-dojo-report-operator
           image: ghcr.io/telekom-mms/docker-trivy-dojo-operator:0.5.7
+          securityContext:
+            runAsUser: 1000
+            runAsNonRoot: true
+            privileged: false
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+            seccompProfile:
+              type: RuntimeDefault
+            capabilities:
+              drop:
+              - ALL
```

Comment:
```suggestion
      securityContext:
          runAsNonRoot: true
          fsGroupChangePolicy: Always
      containers:
        - name: trivy-dojo-report-operator
          image: ghcr.io/telekom-mms/docker-trivy-dojo-operator:0.5.7
          securityContext:
            runAsUser: 1000
            fsGroup: 1001
            privileged: false
            readOnlyRootFilesystem: true
            allowPrivilegeEscalation: false
            seccompProfile:
              type: RuntimeDefault
            capabilities:
              drop:
              - ALL
```

Reply:
podSecurityContext is not a valid field in the deployment spec. I also moved some context between pod and container level